pkg/resource/tls_secret.go

```diff
@@ -94,6 +94,14 @@ func (s *TLSSecret) UpdateCertAndCA(cert, ca []byte, log logr.Logger) error {
 	return err
 }
 
+func (s *TLSSecret) CA() []byte {
```

Comment:
TODO - check which one of these funds are used.  I think one of them is not used.